openresty-build-tools/kong-ngx-build

```diff
@@ -474,6 +481,13 @@ main() {
           OPENRESTY_OPTS+=('--with-luajit-xcflags="-DLUAJIT_USE_VALGRIND -DLUA_USE_ASSERT -DLUA_USE_APICHECK -DLUAJIT_USE_SYSMALLOC"')
         fi
 
+        if [ $PROCESSOR_TYPE == "s390x" ]; then
+          if [ $OPENRESTY_VER == "1.17.8.1" ] || [ "$OPENRESTY_VER" == "1.17.8.2" ]; then
+            # Change the configure file for s390x
+            sed -i '742,785s/.*/#&/;717,726s/.*/#&/' ./configure
+          fi
+        fi
+
```

Comment:
While this is the published fix for s390x, there may be a better fix.  This sed command comments out 2 blocks of code in 'configure' that cause compile errors.  The better solution would introduce Linux on Z as a separate platform in configure and bypass these clauses programmatically instead of commenting them out.

Reply:
Will we revisit the longer term fix? In the meantime it may be advisable to make variables for the OPENRESTY_VER or use a regex to match . . .assuming that 1.17.8.x or 1.17.x need to have this behavior. That may make things more resilent for bug fix or minor version releases?

Reply:
not longer-term.  this needs to be changed now.

Reply:
The configure change actually needs to be made via an openresty pr, which I will open as well.  Meanwhile, an improved check for the openresty version.